src/cryptoadvance/specter/cli/cli_ext.py

```diff
@@ -37,6 +38,12 @@ def ext():
 @ext.command()
 @click.option("--org", "org", default=None, help="Use a specific organsiation")
 @click.option("--ext-id", "ext_id", default=None, help="Use a specific extension id")
+@click.option(
+    "--encrypted-userdata/--no-encrypted-userdata",
+    default=None,
+    help="Whether the user data should be encryted or not",
+)
+@click.option("--version", "tag", default=None, help="Use a specific specter version")
```

Comment:
So should I also remove the option for the user to specify the version while setting up an extension?